src/main/java/main/view/profile/ProfileScene.java

```diff
@@ -11,90 +13,83 @@
-        // TODO Auto-generated method stub
-        return null;
+        return this.scene;
     }
 
     @Override
     protected void updateTop() {
-        root.setTop(super.getMenuBar());
-
+        this.root.setTop(super.getMenuBar());
     }
 
     @Override
     protected void updateBottom() {
-        // TODO Auto-generated method stub
-
+        // nothing to be done here
+        final Pane bottomLayout = getGadgets().createHorizontalPanel();
+        this.root.setBottom(bottomLayout);
     }
 
+    @SuppressWarnings("unchecked")
     @Override
     protected void updateCenter() {
+        final Iterator<List<?>> iter = this.updateables.iterator();
+        final List<String> invAccIds = (List<String>) iter.next();
+        final Iterator<Double> invAccValues = (Iterator<Double>) iter.next().iterator();
+        final List<String> holAccIds = (List<String>) iter.next();
+        final Iterator<Double> holAccValues = (Iterator<Double>) iter.next().iterator();
+
+        final Pane centerLayout = getGadgets().createVerticalPanel();
+
         final Text titleInv = getGadgets().createText("Investment Accounts", TITLE_DIM);
         final ListView<Object> listInvAccs = new ListView<>();
-        getController().getUsrEconomy().getInvestmentAccounts().forEach(acc -> {
-            listInvAccs.getItems().addAll("Name: " + acc.getID(), "Balance: " + acc.getBalance(),
-                    "Invested Balance: " + acc.getInvestedBalance(), "");
+        invAccIds.forEach(id -> {
+            listInvAccs.getItems().addAll(
+                    "Name: " + id,
+                    "Balance: " + this.df.format(invAccValues.next()) + " $",
+                    "Invested Balance: " + this.df.format(invAccValues.next()) + " $",
+                    "");
         });
         final Text titleHol = getGadgets().createText("Holding Accounts", TITLE_DIM);
         final ListView<Object> listHolAccs = new ListView<>();
-        getController().getUsrEconomy().getHoldingAccounts().forEach(acc -> {
-            listHolAccs.getItems().addAll("Name: " + acc.getID(), "Value: " + acc.getTotalValue(), "");
-        });
-
-        final Pane rightLayout = getGadgets().createVerticalPanel();
-        final Button changePassword = getGadgets().createButton("Cambia Password");
-        changePassword.setOnAction(e -> {
-            getController().showPasswordChangeView();
-        });
-        final Button logOut = getGadgets().createButton("Disconnettiti");
-        logOut.setOnAction(e -> {
-            getPrimaryStage().setScene(
-                    new LoginScene(getPrimaryStage(), new MainScene(getPrimaryStage(), getController()).getScene(), getController())
-                            .getScene());
-            getPrimaryStage().centerOnScreen();
+        holAccIds.forEach(id -> {
+            listHolAccs.getItems().addAll(
+                    "Name: " + id,
+                    "Value: " + this.df.format(holAccValues.next()) + " $",
```

Comment:
That could be using Enum class representing dollar, euro, pound, or RMB:)

Reply:
And reads from configuration file for the correct currency, but whatever, good job :)